forestflow/pcross.py

```diff
@@ -0,0 +1,233 @@
+import numpy as np
+from scipy.integrate import simpson
+
+def P1D_Mpc(P3D_Mpc, z, ln_k_perp, kpars, P3D_mode='pol',  **P3D_kwargs):
```

Comment:
What's the difference between this function and the one implemented in model_p3d_arinyo?

Reply:
It is the same except it takes a generic function for P3D_Mpc, so it doesn't have to be called within the ArinyoModel class and P3D can be a function of kperp and kpar or k, mu. The idea is to have this available for calls to the Pcross function with non-Arinyo model power spectra

Reply:
It is great for the time being but in the future, I would like to create a new class containing all these functions. Or to put all functions outside a class. It is clear by now that the old structure is not the best.

Reply:
This is my only comment also, we should reorganize the repo to avoid having two very similar functions.